src/VisualStudio/Core/Def/InheritanceMargin/InheritanceMarginLogger.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Internal.Log;
+
+namespace Microsoft.VisualStudio.LanguageServices.InheritanceMargin
+{
+    internal static class InheritanceMarginLogger
+    {
+        // 1 sec per bucket, and if it takes more than 1 min, then this log is considered as time-out in the last bucket.
+        private static readonly HistogramLogAggregator s_histogramLogAggregator = new(1000, 60000);
```

Comment:
```suggestion
        private static readonly HistogramLogAggregator s_histogramLogAggregator = new(TimeSpan.FromSecons(1), TimeSpan.FromMinutes(1));
```

Reply:
It seems like this is accepting int? https://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Log/HistogramLogAggregator.cs,20